examples/bh_threaded.c

```diff
@@ -13,6 +13,11 @@
 #include <linux/gpio.h>
 #include <linux/delay.h>
 #include <linux/interrupt.h>
+#include <linux/version.h>
+
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 10, 0)
+#define GPIO_NO_REQUEST_ARRAY
```

Comment:
Rename to `NO_GPIO_REQUEST_ARRAY`.

Reply:
I have fixed it.

Reply:
You don't have to put the message "I have fixed it." Instead, simply click "Resolve conversation" via GitHub web interface once you complete.